bar.m

```diff
@@ -882,7 +853,7 @@ static int userdata_gc(lua_State* L) {
 // };
 
 int luaopen_hs__asm_undocumented_touchbar_bar(lua_State* L) {
-    [NSApplication xxx_hammerspoon_SwizzleCrashOnException] ;
+    [NSFunctionRow xxx_hammerspoon_SwizzleMarkActiveFunctionRowsAsDimmed];
```

Comment:
Oof, surprised that compiled... are you still building this as a separate module or wrapped up in CommandPost? I've occasionally found differences in which approach throws what errors during compile.

Reply:
I'm currently just forking this repo, and using your build script. I then manually copy the compiled files into CommandPost.